### PR TITLE
updated pileup tutorial and supporting documents for genotype example

### PR DESCRIPTION
**Pull request by abhim00**

### CODE REVIEW

File: docs/jupyterWidgets/usage.rst [hunk 1]
```diff
@@ -57,11 +57,8 @@ This example shows how to visualize variants through a Jupyter widget.
 
 .. code:: python
 
-    variantsJson = pd.read_json("./data/variants.ga4gh.chr1.10000-11000.json")
-    GA4GHVariantJson = variantsJson.to_json()
-
     # make variant track
-    tracks=[Track(viz="variants", label="my Variants", source=pileup.sources.GA4GHVariantJson(GA4GHVariantJson))]
+    tracks=[Track(viz="variants", label="my Variants", source=pileup.sources.VcfDataSource("./data/genodata.v3.vcf"))]
```
Comment (akmorrow13) [inline docs/jupyterWidgets/usage.rst]: Can you make this a fake path, if it does not exist, ie <path_to_vcf.vcf>?

File: docs/jupyterWidgets/usage.rst [hunk 2]
```diff
@@ -90,3 +86,19 @@ This example shows how to visualize features through a Jupyter widget.
...
+
+Genotype Example
+---------------
+
+This example shows how to visualize genotypes through a Jupyter widget.
+
+.. code:: python
+
+    # make genotype track
+    tracks=[Track(viz="features", label="my Features", source=pileup.sources.VcfDataSource(./data/genodata.v3.vcf))]
```
Comment (akmorrow13) [inline docs/jupyterWidgets/usage.rst]: see comment above. Also, I dont think this would work because the filepath is not in quotes

Comment (akmorrow13) [reply docs/jupyterWidgets/usage.rst]: This is a feature visualization, not genotypes

File: mango-viz/examples/pileup-tutorial.ipynb [hunk 3]
```diff
@@ -170,11 +171,49 @@
...
+       "A Jupyter Widget"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# make genotype track\n",
+    "tracks=[Track(viz=\"genotypes\", label=\"my genotypes\", source=pileup.sources.VcfDataSource(\"./data/genodata.v3.vcf\"))]\n",
```
Comment (akmorrow13) [inline mango-viz/examples/pileup-tutorial.ipynb]: I don't see any code modifying the viz genotypes. (see https://github.com/bigdatagenomics/mango/blob/master/mango-viz/bdgenomics/mango/pileup/sources.py#L218 ) Did you forget to push this?

Comment (abhim00) [reply mango-viz/examples/pileup-tutorial.ipynb]: Yes. I forgot to push this.. will do that now.

File: docs/jupyterWidgets/usage.rst (hunk 2 above)
Comment (abhim00) [reply docs/jupyterWidgets/usage.rst]: will fix this... was an oversight

**Comment by abhim00** [comment]
I made the changes, hope this works.

File: docs/jupyterWidgets/usage.rst [hunk 4]
```diff
@@ -90,3 +86,19 @@ This example shows how to visualize features through a Jupyter widget.
...
+
+Genotype Example
+---------------
+
+This example shows how to visualize genotypes through a Jupyter widget.
+
+.. code:: python
+
+    # make genotype track
+    tracks=[Track(viz="genotypes", label="my Genotypes", source=pileup.sources.VcfDataSource("/path/to_vcf.vcf"))]
```
Comment (akmorrow13) [inline docs/jupyterWidgets/usage.rst]: "/path/to_vcf.vcf" -> "<path_to_file>/my_vcf.vcf"

**Comment by akmorrow13** [comment]
Thanks @abhim00 just one minor change other than that looks good!

**Comment by akmorrow13** [comment]
Jenkins, test this please.

**Comment by abhim00** [comment]
fixed that issue !

**Comment by akmorrow13** [comment]
Jenkins, retest this please.

**Comment by akmorrow13** [comment]
Thanks @abhim00 !

**Pull request merged**